test/nft_upload_test.py

```diff
@@ -25,4 +25,11 @@
 ...     time.sleep(10)
 >>> status
 True
+>>> while(status == False and counter<try_limit):
```

Comment:
@sadrasabouri Please merge this section with the previous `nft_storage` upload(to prevent multi uploads)